### PR TITLE
doc(ipv6.md): update for increased dual-stack support

### PR DESCRIPTION
**Pull request by aauren**
@mrueg 

This updates the IPv6 documentation on the default branch to include up-to-date information on the current status of dual-stack support integration.

This goes hand-in-hand with the v2.0.0-rc1 pre-release that was just made.

### CODE REVIEW

File: docs/ipv6.md [hunk 1]
```diff
@@ -1,114 +1,175 @@
...
-should get the protocol from a global setting for ipv6-only and later
-from some attribute in the service object.
+The new `kube-router.io/pod-cidrs` annotation is a comma-separated list of CIDRs and can hold either IPv4 or IPv6 CIDRs
+in string form.
 
-Since the same `gobgp` is used for the CNI and the router functions is
-may be fairly simple to implement ipv6 support.
+It should be noted, that until `kube-router.io/pod-cidr` is fully removed, at some point in the future, it will still
+be preferred over the `kube-router.io/pod-cidrs` annotation in order to preserve as much backwards compatibility as
+possible. Until `kube-router.io/pod-cidr` has been fully retired, user's the use the old annotation will get a warning
```
Comment (mrueg) [inline docs/ipv6.md:124]:
```suggestion
possible. Until `kube-router.io/pod-cidr` has been fully retired, user's that use the old annotation will get a warning
```

File: docs/ipv6.md [hunk 2]
```diff
@@ -1,114 +1,175 @@
...
+`kube-router.io/pod-cidrs` annotation. Since kube-router currently only updates node annotations at start and not as
+they are updated, this is a safe change to make before updating kube-router.
 
+If neither annotation is specified, kube-router will use the `PodCIDRs` field of the Kubernetes node spec which is
+populated by the `kube-controller-manager` as part of it's `--allocate-node-cidrs` functionality. This should be a sane
+default for most users of kube-router.
 
+### CNI Now Accepts Multiple Pod Ranges
 
+Now that kube-router supports dual-stack, it now also supports multiple ranges in the CNI file. While kube-router will
```
Comment (mrueg) [inline docs/ipv6.md:137]:
```suggestion
Now that kube-router supports dual-stack, it also supports multiple ranges in the CNI file. While kube-router will
```

File: docs/ipv6.md [hunk 3]
```diff
@@ -1,114 +1,175 @@
...
+* CNI
+* Router / BGP (`--run-router`)
+* Network Policies (`--run-firewall`)
 
+## How Can You Help?
 
-## Current status (Thu Oct 11 2018)
+The work to make dual-stack support workable in kube-router has involved a lot of work and several large contributions
+to the code base. If you have access to a development or staging environment where you would be willing to run the
+latest v2.0.0 release candidate and give us feedback, it would be much appreciated!
```
Comment (mrueg) [inline docs/ipv6.md:37]: Maybe add how to provide feedback (e.g. slack, issue, etc.)

File: docs/ipv6.md [hunk 4]
```diff
@@ -1,114 +1,175 @@
...
-  "isDefaultGateway": true,
-  "isGateway": true,
-  "name": "ekvm",
-  "type": "bridge"
-}
-```
+## Roadmap
 
-This means that pod's gets assigned ipv6 addresses.
+The next big item to add to make kube-router fully dual-stack functional will be the **Proxy (`--run-service-proxy`)**
```
Comment (mrueg) [inline docs/ipv6.md:51]: Can we create an issue for this and reference it here so users can track it?

**Review by mrueg — APPROVED** [review]

File: docs/ipv6.md (hunk 3 above)
Comment (aauren) [reply docs/ipv6.md:37]: Added in the paragraph below.

File: docs/ipv6.md (hunk 4 above)
Comment (aauren) [reply docs/ipv6.md:57]: Good idea! I've added a new issue template in this PR.

File: docs/ipv6.md (hunk 1 above)
Comment (aauren) [reply docs/ipv6.md]: Fixed

File: docs/ipv6.md (hunk 2 above)
Comment (aauren) [reply docs/ipv6.md]: Fixed

File: docs/ipv6.md [hunk 5]
```diff
@@ -121,7 +127,7 @@ in string form.
 
 It should be noted, that until `kube-router.io/pod-cidr` is fully removed, at some point in the future, it will still
 be preferred over the `kube-router.io/pod-cidrs` annotation in order to preserve as much backwards compatibility as
-possible. Until `kube-router.io/pod-cidr` has been fully retired, user's the use the old annotation will get a warning
+possible. Until `kube-router.io/pod-cidr` has been fully retired, user's that use the old annotation will get a warning
```
Comment (mrueg) [inline docs/ipv6.md:130]:
```suggestion
possible. Until `kube-router.io/pod-cidr` has been fully retired, users that use the old annotation will get a warning
```

File: docs/ipv6.md (hunk 4 above)
Comment (mrueg) [reply docs/ipv6.md:57]: I was thinking about creating an issue for the missing --run-service-proxy functionality, that users can track

Comment (aauren) [reply docs/ipv6.md:57]: Ahh... I see now that I originally misunderstood this.

Here is a tracking issue: https://github.com/cloudnativelabs/kube-router/issues/1432

**Comment by mrueg** [comment]
Lgtm

**Pull request merged**